<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <!-- Barra superior con logo -->
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>                        Imagenología Médica</string>
   </property>
   <!-- Estilo -->
   <property name="styleSheet">
    <string>background-color: #32a985; color: #d6eee7; font-size: 40px; font-weight: bold; text-align: center;</string>
   </property>
  </widget>
  <!-- Eslogan -->
  <widget class="QLabel" name="slogan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>987</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>                              Explorador de imágenes de resonancia magnética y tomografía computarizada</string>
   </property>
   <!-- Estilo -->
   <property name="styleSheet">
    <string>background-color: #32a985; color: #0a221b; font-size: 18px; text-align: center;</string>
   </property>
  </widget>
  <!-- Etiquetas y Widgets -->
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sagital</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Axial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Coronal</string>
   </property>
  </widget>
  <widget class="QWidget" name="eje_sagital" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="eje_coronal" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="eje_axial" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>271</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
<widget class="QPushButton" name="regresar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar Sesión</string>
   </property>
   <!-- Estilo -->
   <property name="styleSheet">
    <string>background-color: #32a985; color: #d6eee7; border: none; border-radius: 7.5px; text-align: center;</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="info_im">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Información del paciente </string>
   </property>
   <!-- Estilo -->
   <property name="styleSheet">
    <string>background-color: #32a985; color: #d6eee7; border: none; border-radius: 7.5px; text-align: center;</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor_actual">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="valor_maximo">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>420</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>430</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
